### PR TITLE
Stock Status by Product UI fixes

### PR DESCRIPTION
**Pull request by pr33thi**
This fixes a couple issues in graphing, tangential to this ticket:
https://manage.dimagi.com/default.asp?277444#1502313

### CODE REVIEW

File: corehq/apps/reports/templates/reports/partials/graphs/multibar_chart.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% load hq_shared_tags %}
 
-<div class="charts-pie-chart"
+<div class="charts-multibar-chart"
```
Comment (emord) [inline corehq/apps/reports/templates/reports/partials/graphs/multibar_chart.html]: lol is this why that report wasn't showing anything?

**Review by emord — APPROVED** [review]

File: corehq/apps/reports/templates/reports/partials/graphs/multibar_chart.html (hunk 1 above)
Comment (pr33thi) [reply corehq/apps/reports/templates/reports/partials/graphs/multibar_chart.html]: yep :D

Comment (jmtroth0) [reply corehq/apps/reports/templates/reports/partials/graphs/multibar_chart.html]: 😂

File: corehq/apps/reports/graph_models.py [hunk 2]
```diff
@@ -12,6 +12,11 @@ def __init__(self, label=None, format=None, dateFormat=None):
         self.format = format
         self.dateFormat = dateFormat
 
+    def to_json(self):
+        return {'label': self.label,
+                'format': self.format,
+                }
```
Comment (jmtroth0) [inline corehq/apps/reports/graph_models.py]: is dateFormat not necessary to pass down to the JS?

Comment (pr33thi) [reply corehq/apps/reports/graph_models.py]: True, it wasn't necessary to fix this, but I added it here:
https://github.com/dimagi/commcare-hq/pull/20884

**Pull request merged**